derp/derp_server.go

```diff
@@ -1816,8 +1811,8 @@ func (c *sclient) sendLoop(ctx context.Context) error {
 	}
 }
 
-func (c *sclient) setWriteDeadline() {
-	d := writeTimeout
+func (c *sclient) setWriteDeadline() error {
```

Comment:
The failure modes of `SetWriteDeadline()` are acceptable when taken in context, i.e. the i/o ops will subsequently fail. We can continue to swallow this error.